static/js/hotkey.js

```diff
@@ -781,9 +784,6 @@ export function process_hotkey(e, hotkey) {
         case "star_deprecated":
             ui.maybe_show_deprecation_notice("*");
             return true;
-        case "copy_with_c":
```

Comment:
Oh! This was wrongly mapped as a compose box hotkey.

Reply:
Yes, but I think this actually belongs after the `message_lists.current.empty` check, since it'll do nothing useful unless message content is selected.